samples/hello-js/build.gradle.kts

```diff
@@ -0,0 +1,45 @@
+import org.gradle.kotlin.dsl.kotlin
+import org.jetbrains.kotlin.gradle.tasks.Kotlin2JsCompile
+
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath(kotlin("gradle-plugin"))
+    }
+}
+
+apply {
+    plugin("kotlin2js")
+}
+
+dependencies {
+    "compile"(kotlin("stdlib-js"))
+}
+
+repositories {
+    jcenter()
+}
+
+tasks.withType<Kotlin2JsCompile> {
+    kotlinOptions.outputFile = "$projectDir/web/output.js"
+    kotlinOptions.moduleKind = "plain"
+    kotlinOptions.sourceMap = true
+}
+
+val build by tasks
+build.doLast {
```

Comment:
:x: this should be modeled as a separate task and wired to `assemble` instead of `build`, e.g. :

```kotlin
val stageWeb by tasks.creating
stageWeb.doLast {
    // ...
}
val assemble by tasks
assemble.dependsOn(stageWeb)
```